docs/eloquent-models/model-class.txt

```diff
@@ -194,6 +194,24 @@ type, to the Laravel ``datetime`` type.
    :emphasize-lines: 9-11
    :dedent:
 
+.. tip:: Casts in Laravel 11
+
+   In Laravel 11, you can define a ``casts()`` method to specify data type conversions
+   instead of using the ``$casts`` attribute. The following code performs the same
+   conversion as the preceding example by using a ``casts()`` method:
+
+   .. code-block:: php
+
+      protected function casts(): array
+      {
+         return [
+            'discovery_dt' => 'datetime',
+         ];
+      }
+   
+   To learn more, see the `Model Casts are moving to methods in Laravel 11 <https://laravel-news.com/model-casts>`__
+   article on the Laravel News website.
```

Comment:
This is now the default in the Laravel documentation: https://laravel.com/docs/11.x/eloquent-mutators#attribute-casting

I think we should link to the official docs instead of a blogpost.